controllers/progressiverollout_controller.go

```diff
@@ -109,21 +110,13 @@ func (r *ProgressiveRolloutReconciler) Reconcile(ctx context.Context, req ctrl.R
 			}
 		}
 
-		if scheduler.IsStageFailed(apps, stage) {
+		if scheduler.IsStageFailed(apps) {
 			// TODO: updated status
 			r.Log.Info("stage failed")
 			return ctrl.Result{}, nil
 		}
 
-		if scheduler.IsStageComplete(apps, stage) {
-			// TODO: update status
-			r.Log.Info("stage completed")
-		} else {
-			// TODO: update status
-			r.Log.Info("stage in progress")
-			// Stage in progress, we reconcile again until the stage is completed or failed
-			return ctrl.Result{Requeue: true}, nil
-		}
+		r.Log.Info("stage completed")
```

Comment:
This is an 'odd change' - it's perfectly backwards compatible with what's in `main` as `IsStageComplete(apps, stage)` always returned true. But ensures backwards compatibility for tests as `IsStageComplete` has an actual body now and tests would fail otherwise.

It'll be promptly changed again in #60.